test/db/analysis/java

```diff
@@ -1,189 +1,164 @@
-NAME=java prototypes in json
-FILE=bins/java/jdwpexeccmd.class
-CMDS=java prototypes j
-EXPECT=<<EOF
-{"access_flags":33,"is_public":true,"is_final":false,"is_super":true,"is_interface":false,"is_abstract":false,"is_synthetic":false,"is_annotation":false,"is_enum":false,"name":"JDWPExecCmd","super":"java/lang/Object","interfaces":[],"methods":[{"access_flags":1,"is_method":true,"is_native":false,"is_synthetic":false,"is_private":false,"is_public":true,"is_static":false,"is_protected":false,"is_super":false,"addr":825,"offset":803,"class_name":"java/lang/Object","signature":"()V","name":"<init>","fq_name":"java/lang/Object.<init>.()V","prototype":"public void <init> ()"},{"access_flags":9,"is_method":true,"is_native":false,"is_synthetic":false,"is_private":false,"is_public":true,"is_static":true,"is_protected":false,"is_super":false,"addr":868,"offset":846,"class_name":"JDWPExecCmd","signature":"(Ljava/lang/String;)Ljava/lang/String;","name":"run","fq_name":"JDWPExecCmd.run.(Ljava/lang/String;)Ljava/lang/String;","prototype":"public static java.lang.String run (java.lang.String)"}],"fields":[],"imports":["java.lang.StringBuilder","java.io.BufferedReader","java.io.InputStreamReader","java.lang.Exception","JDWPExecCmd","java.lang.Object","java.lang.String","java.lang.Process","java.io.BufferedReader","java.lang.Exception","java.lang.Runtime","java.lang.Process","java.lang.String"]}
-EOF
-RUN
 
-NAME=java prototypes in ascii
-FILE=bins/java/jdwpexeccmd.class
-CMDS=java prototypes a
-EXPECT=<<EOF
-import java.lang.StringBuilder;
-import java.io.BufferedReader;
-import java.io.InputStreamReader;
-import java.lang.Exception;
-import JDWPExecCmd;
-import java.lang.Object;
-import java.lang.String;
-import java.lang.Process;
-import java.io.BufferedReader;
-import java.lang.Exception;
-import java.lang.Runtime;
-import java.lang.Process;
-import java.lang.String;
-
-class JDWPExecCmd { // @0x0000
```

Comment:
do these addresses still make sense? It seems you removed them.

Reply:
no. if you want to know the addresses, please use the ic command not that.